python/py-mitmproxy/Portfile

```diff
@@ -24,57 +24,56 @@ long_description    \
 
 homepage            http://mitmproxy.org/
 
-checksums           md5     954b5796ab36c6d7e5dad5775e7178a3 \
-                    rmd160  6d28eec42f4e410775a8d394f6c12232652df691 \
-                    sha256  9b1460be1cf16ceb8b89c11f8593ad2de16ec9ce238964f26eb36e1bcfb13f24
+checksums           md5     d49a024cf743ce3cc1ae4ba4a613166e \
+                    rmd160  5184bd752cae6ffbb61a11c4516e1ceb0ac1973a \
+                    sha256  4b67f68dd488af6ccab4f534124e63ad583776922dc004166212254a33b2a07b
 
-python.versions     27
```

Comment:
Is there any special reason for removing the port `py27-mitmproxy`? We usually remove old python versions, but basically never those for py27 unless a module is no longer compatible with python 2. If the intention was in fact to remove it, it could perhaps be `replaced_by` `py36-mitmproxy`, even though I'm not absolutely sure about that (if one is using python2.7, that won't help at all). I didn't see any port depending on this module, so I cannot really judge.

What's now with py-netlib which also contains the py36 subport? Should that port be removed and `replaced_by` py-mitmproxy (judging from comments, I didn't really check anything myself)?

Reply:
The commit message said "mitmproxy is now Python 3 only". If upstream no longer supports it, that should be reason enough to drop the py27-* subport. I don't think a `replaced_by` would help anyone. For those who already have it, let them keep using the old version, I see no harm with that.

Reply:
OK, thanks for explanation, I was only going through the changes themselves. I agree that `replaced_by` doesn't really help anyone. The question about the status of `py-netlib` still stands.

Reply:
netlib is renamed when merged into mitmproxy (https://github.com/mitmproxy/mitmproxy/pull/1637), so technically there's no problem if a user wants both py36-mitmproxy and py36-netlib activated. I just tried it and I can import ```netlib``` from simple scripts running with ```python3.6``` and run mitmproxy-3.6 at the same time. I don't know who really needs standalone netlib, though.